### PR TITLE
model: add env var to turn on v3 model

### PR DESCRIPTION
**Pull request by shalins**
### Description of Changes
* Adds an env var to switch to new v3 spacebar model.
* Conditionally changes the max boundary to be much larger.

### Testing Procedure
* Tested with and without the flag, both in env.local and inline (`REACT_APP_EXPERIMENTAL_MODEL=true yarn localStart`).

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@ import log from 'loglevel';
 
 import { Bucket, resolveStoragePath, resolveStoragePaths } from './utils/io';
 
+// TODO(SHALIN): Remove once the new model is deployed.
+const experimentalModel = process.env.REACT_APP_EXPERIMENTAL_MODEL === 'true';
```
Comment (samjgorman) [inline src/common/constants.ts:6]:
```suggestion
const isExperimentalModelEnabled = process.env.REACT_APP_EXPERIMENTAL_MODEL === 'true';
```

File: src/common/constants.ts [hunk 2]
```diff
@@ -36,7 +39,7 @@ export abstract class TerrainConstants {
 
 export abstract class Objects {
   static readonly barPath: string = resolveStoragePath(
-    'modded_cafe/modded_cyberbar.gltf',
+    `${experimentalModel ? 'v2/spacebar.gltf' : 'modded_cafe/modded_cyberbar.gltf'}`,
```
Comment (samjgorman) [inline src/common/constants.ts:42]:
```suggestion
    `${isExperimentalModelEnabled? 'v2/spacebar.gltf' : 'modded_cafe/modded_cyberbar.gltf'}`,
```

File: src/common/constants.ts [hunk 3]
```diff
@@ -92,13 +95,13 @@ export abstract class Boundary {
 
   //HACK(Kanyes) Hard-code a player bounding box for the 5/3 demo.
   // In the future we should be inferring these values from the glTFs directly
-  static readonly boundaryMaxX = 13;
+  static readonly boundaryMaxX = experimentalModel ? 50 : 13;
```
Comment (samjgorman) [inline src/common/constants.ts:98]:
```suggestion
  static readonly boundaryMaxX = isExperimentalModelEnabled ? 50 : 13;
```

File: src/common/constants.ts [hunk 4]
```diff
@@ -92,13 +95,13 @@ export abstract class Boundary {
...
 
-  static readonly boundaryMinX = -30;
+  static readonly boundaryMinX = experimentalModel ? -50 : -30;
```
Comment (samjgorman) [inline src/common/constants.ts:100]:
```suggestion
  static readonly boundaryMinX = isExperimentalModelEnabled ? -50 : -30;
```

File: src/common/constants.ts [hunk 5]
```diff
@@ -92,13 +95,13 @@ export abstract class Boundary {
...
 
-  static readonly boundaryMaxZ = 10.5;
+  static readonly boundaryMaxZ = experimentalModel ? 50 : 10.5;
```
Comment (samjgorman) [inline src/common/constants.ts:102]:
```suggestion
  static readonly boundaryMaxZ = isExperimentalModelEnabled ? 50 : 10.5;
```

File: src/common/constants.ts [hunk 6]
```diff
@@ -92,13 +95,13 @@ export abstract class Boundary {
...
 
-  static readonly boundaryMinZ = -5.5;
+  static readonly boundaryMinZ = experimentalModel ? -50 : -5.5;
```
Comment (samjgorman) [inline src/common/constants.ts:104]:
```suggestion
  static readonly boundaryMinZ = isExperimentalModelEnabled ? -50 : -5.5;
```

**Review by samjgorman — APPROVED** [review]
Looks good! Let's just make the var more explicit that it's a bool

File: src/common/constants.ts (hunk 6 above)
Comment (shalins) [reply src/common/constants.ts]: updated everywhere

**Pull request merged**